examples/landmark_uncertainty/README.md

```diff
@@ -0,0 +1,369 @@
+
+Default:
+`_C.PIPELINE.COMPARE_Q_UNCERTAINTY_ERROR_PAIRS` = [["E-MHA", "E-MHA Error", "E-MHA Uncertainty"]]
+
+#### `PIPELINE.COMBINE_MIDDLE_BINS`
+
+A boolean indicating whether to combine the middle quantile bins into a single bin.
+
+Default:
+`_C.PIPELINE.COMBINE_MIDDLE_BINS` = False
+
+#### `PIPELINE.PIXEL_TO_MM_SCALE`
+
+A float specifying the scale factor to convert pixel units to millimeter units.
+
+Default:
+`_C.PIPELINE.PIXEL_TO_MM_SCALE` = 1.0
+
+#### `PIPELINE.IND_LANDMARKS_TO_SHOW`
+
+A list of landmark indices to show individually. A value of -1 means show all landmarks individually, and an empty list means show none.
+
+Default:
+`_C.PIPELINE.IND_LANDMARKS_TO_SHOW` = [-1]
+
+#### `PIPELINE.SHOW_IND_LANDMARKS`
+
+A boolean indicating whether to show results from individual landmarks.
+
+Default:
+_C.`PIPELINE.SHOW_IND_LANDMARKS` = True
+
+<br/><br/>
+
+### C) Visualisation: IM_KWARGS, MARKER_KWARGS, WEIGHT_KWARGS
+The following are the configuration options related to visualization and plotting.
+
+#### `IM_KWARGS.CMAP`
+
+The color map to use for the image.
+
+Default:
+`_C.IM_KWARGS.cmap` = "gray"
+
+#### `MARKER_KWARGS.MARKER`
+
+The marker style to use for the landmark points.
+
+Default:
+`_C.MARKER_KWARGS.marker` = "o"
+
+#### `MARKER_KWARGS.MARKERFACECOLOR`
+
+The face color to use for the landmark points.
+
+Default:
+_C.MARKER_KWARGS.markerfacecolor = (1, 1, 1, 0.1)
+
+#### `MARKER_KWARGS.MARKEREDGEWIDTH`
+
+The edge width to use for the landmark points.
+
+Default:
+`_C.MARKER_KWARGS.markeredgewidth` = 1.5
+
+#### `MARKER_KWARGS.MARKEREDGECOLOR`
+
+The edge color to use for the landmark points.
+
+Default:
+`_C.MARKER_KWARGS.markeredgecolor` = "r"
+
+#### `WEIGHT_KWARGS.MARKERSIZE`
+
+The size to use for the weights of the landmark points.
+
+Default:
+`_C.WEIGHT_KWARGS.markersize` = 6
+
+#### `WEIGHT_KWARGS.ALPHA`
+
+The transparency to use for the plots.
+
+Default:
+`_C.WEIGHT_KWARGS.alpha` = 0.7
+
+<br/><br/>
+
+
+### D) BOXPLOT
+The following are the configuration options related to the box plot.
+
+
+#### `BOXPLOT.SAMPLES_AS_DOTS`
+
+A boolean indicating whether to show the samples as dots on the box plot (can be expensive if many landmarks.).
+
+Default:
+`_C.BOXPLOT.SAMPLES_AS_DOTS` = True
+
+#### `BOXPLOT.ERROR_LIM`
+
+The error limit to use for the box plot.
+
+Default:
+`_C.BOXPLOT.ERROR_LIM` = 64
+
+#### `BOXPLOT.SHOW_SAMPLE_INFO_MODE`
+
+The mode for showing sample information on the box plot. The available modes are "None", "All", and "Average".
+
+Default:
+_C.BOXPLOT.SHOW_SAMPLE_INFO_MODE = "Average"
+
+<br/><br/>
+
+
+### E) OUTPUT
+
+The following are miscellaneous configuration options.
+
+#### `OUTPUT.SAVE_FOLDER`
+
+The folder where the experiment results will be saved.
+
+Default:
+`_C.OUTPUT.SAVE_FOLDER` = "../../../results/"
```

Comment:
See the comment at the top. We should have a standardised directory for outputs/results.